project/MiMa.scala

```diff
@@ -17,10 +17,230 @@ object MiMa extends AutoPlugin {
+      ProblemFilters.exclude[MissingClassProblem]("akka.cluster.routing.AdaptiveLoadBalancingRouter"),
+      // class akka.cluster.routing.ClusterRouterConfig does not have a correspondent in new version
+      ProblemFilters.exclude[MissingClassProblem]("akka.cluster.routing.ClusterRouterConfig"),
+      // deprecated method this(Int,java.lang.String,Boolean,java.lang.String)Unit in class akka.cluster.routing.ClusterRouterGroupSettings does not have a correspondent with same parameter signature among (Int,java.lang.Iterable,Boolean,java.lang.String)Unit, (Int,scala.collection.immutable.Seq,Boolean,scala.Option)Unit
+      ProblemFilters.exclude[IncompatibleMethTypeProblem]("akka.cluster.routing.ClusterRouterGroupSettings.this"),
+      // deprecated method this(Int,java.lang.String,Boolean,scala.Option)Unit in class akka.cluster.routing.ClusterRouterGroupSettings does not have a correspondent with same parameter signature among (Int,java.lang.Iterable,Boolean,java.lang.String)Unit, (Int,scala.collection.immutable.Seq,Boolean,scala.Option)Unit
+      ProblemFilters.exclude[IncompatibleMethTypeProblem]("akka.cluster.routing.ClusterRouterGroupSettings.this"),
+      ProblemFilters.exclude[MissingMethodProblem]("akka.testkit.TestKit.dilated"),
+      
+      
+      // changed internals
+      ProblemFilters.exclude[MissingMethodProblem]("akka.actor.ActorSystem.terminate"),
+      ProblemFilters.exclude[MissingMethodProblem]("akka.actor.ActorSystem.whenTerminated"),
+      ProblemFilters.exclude[MissingMethodProblem]("akka.actor.ExtendedActorSystem.logFilter"),
+      // the type hierarchy of class akka.actor.ChildActorPath has changed in new version. Missing types {akka.actor.ActorPath}
+      ProblemFilters.exclude[MissingTypesProblem]("akka.actor.ChildActorPath"),
+      // the type hierarchy of class akka.actor.RootActorPath has changed in new version. Missing types {akka.actor.ActorPath}
+      ProblemFilters.exclude[MissingTypesProblem]("akka.actor.RootActorPath"),
+      // declaration of trait akka.actor.ActorPath has changed to class akka.actor.ActorPath in new version; changing trait to class breaks client code
+      ProblemFilters.exclude[IncompatibleTemplateDefProblem]("akka.actor.ActorPath"),
+      ProblemFilters.exclude[MissingMethodProblem]("akka.actor.LocalActorRefProvider.terminationPromise"),
+      ProblemFilters.exclude[MissingClassProblem]("akka.actor.UntypedActorFactoryConsumer"),
+      ProblemFilters.exclude[MissingMethodProblem]("akka.actor.FSM#State.copy"),
+      ProblemFilters.exclude[MissingMethodProblem]("akka.actor.ActorSystemImpl.terminationFuture"),
+      ProblemFilters.exclude[MissingMethodProblem]("akka.actor.IndirectActorProducer.UntypedActorFactoryConsumerClass"),
+      FilterAnyProblem("akka.actor.ActorSystemImpl"),
+      ProblemFilters.exclude[MissingMethodProblem]("akka.pattern.AskSupport.ask"),
+      FilterAnyProblem("akka.pattern.PromiseActorRef"),
```

Comment:
strange, I did fix PromiseActorRef, I'll investigate why that is not included here